src/witan/models/load_data.clj

```diff
@@ -45,7 +45,7 @@
   (make-ordered-ds-schema [[:gss-code s/Str] [:district s/Str] [:sex s/Str] [:age s/Int]
                            [:var s/Str] [:year s/Int] [:estimate s/Num]]))
 
-(def HistPopnEst
+(def BasePopn
```

Comment:
I'm not sure. Load-data is used by the test ns so far to apply coercion on the uploaded testing data.
Ultimately we'll have several code ns w/ schemas. Do we want to require those ns in the load-data ns?